DemoqaProject/tests/ElementsTest/RadioButtonTest.cs

```diff
@@ -19,14 +19,16 @@ public void ClickAndCheck()
             Assert.That(radioButton.checkState.Text, Does.Contain("Impressive"));
         }
 
+        [Test]
         public void CheckNoButton()
         {
             HomePage homePage = new HomePage();
             homePage.NavigateToElementPage();
             RadioButton radioButton = new RadioButton();
             radioButton.NavigateToRadioButton();
             
-            Assert.IsTrue(radioButton.noRadio.Enabled);
+            radioButton.CheckNoRadio();
+            Assert.IsFalse(radioButton.noRadio.Selected);
```

Comment:
It's not what I've asked for. You should check 'disabled' element, not unselected.